### PR TITLE
Added llama2-13b-chat-v1 on bedrock

### PR DESCRIPTION
**Pull request by sungeuns**
- **Description:** Added `llama2-13b-chat-v1` for Bedrock client
  - **Issue:** #13316
  - **Dependencies:** None

Please make sure your PR is passing linting and testing before submitting. Run `make format`, `make lint` and `make test` to check this locally.

See contribution guidelines for more information on how to write/run tests, lint, etc: 
https://github.com/langchain-ai/langchain/blob/master/.github/CONTRIBUTING.md

If you're adding a new integration, please include:
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/extras` directory.

If no one reviews your PR within a few days, please @-mention one of @baskaryan, @eyurtsev, @hwchase17.

### CODE REVIEW

**Comment by jonathancaevans** [comment]
Looks like you beat me to it, exactly the same changes I'd staged! Looks good to me +1! @3coins @hwchase17

**Comment by diegogsoares** [comment]
+1

**Review by 3coins — APPROVED** [review]
@sungeuns 
Thanks for adding Llama2 support. LGTM! 🚀

**Comment by prakass1** [comment]
This was fast, I did the exact changes :)

**Comment by WouterDurnez** [comment]
Open question: shouldn't this technically (also) be in `BedrockChat`, since it's a chat model? https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/chat_models/bedrock.py

**Comment by WouterDurnez** [comment]
Fyi, we just created a pull request to add it, ourselves: https://github.com/langchain-ai/langchain/pull/13403

**Comment by prakass1** [comment]
> Open question: shouldn't this technically (also) be in `BedrockChat`, since it's a chat model? https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/chat_models/bedrock.py

Yes it should be

**Comment by marcio-ch** [comment]
Hi there, when can we get a new release with this change? I have been using a manually updated version of langchain for that, will feel more comfortable when we get the official update. Thanks!

**Comment by 3coins** [comment]
@baskaryan 
Can you help with merging this.

File: libs/langchain/langchain/llms/bedrock.py [hunk 1]
```diff
@@ -80,7 +81,7 @@ def prepare_input(
         input_body = {**model_kwargs}
         if provider == "anthropic":
             input_body["prompt"] = _human_assistant_format(prompt)
-        elif provider == "ai21" or provider == "cohere":
+        elif provider == "ai21" or provider == "cohere" or provider == "meta":
```
Comment (3coins) [inline libs/langchain/langchain/llms/bedrock.py:84]: Suggesting a minor update to simplify the check. Accepting this should also kick in the CI. 
```suggestion
        elif provider in ["ai21", "cohere", "meta"]:
```

**Comment by baskaryan** [comment]
believe all functionality was covered in #13403 so closing this pr, let me know if i'm missing something!

**Pull request closed without merging**